radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -628,7 +639,7 @@ if(PCB STREQUAL XLITE OR PCB STREQUAL XLITES OR PCBREV STREQUAL COMMANDO8)
     targets/common/arm/stm32/sticks_pwm_driver.cpp)
 endif()
 
-if(PCBREV STREQUAL MT12)
+if(PCBREV STREQUAL MT12 OR PCBREV STREQUAL T12MAX)
```

Comment:
Is this right? Didn't think the T12MAX had WS2812s?

Reply:
Right, it was planned, but I think last decision was to disable it since their was question on 5V power rail with power hungry RGB leds, so changing it (and will change it to an option in main). The pin is there tho